fda/basis.py

```diff
@@ -1514,7 +1484,7 @@ def shift(self, shifts, ext="default", tfine=[], **kwargs):
             domain = self.domain_range
 
         # Matrix of shifted values
-        _data_matrix = self.evaluate_shifted(tfine, shifts, ext=extrapolation)
+        _data_matrix = self.evaluate_shifted(tfine, shifts, ext=extrapolation.value)
```

Comment:
Why not `extrapolation` but its value?

Reply:
It's a little typo, I will fix it soon. I would like talking about if it's necessary evaluate_shifted to be a public function. If it would be a private function, the error control of the arguments could be omitted in order to improve performance.

Reply:
Fixed in 9c17e4d.